test/lit/wasm-split/minimized-exports.wast

```diff
@@ -1,23 +1,34 @@
-;; RUN: wasm-split %s --keep-funcs=foo,bar --export-prefix='%' -o1 %t.1.wasm -o2 %t.2.wasm
+;; RUN: wasm-split %s --keep-funcs=foo,bar --export-prefix='%' -o1 %t.1.wasm -o2 %t.2.wasm --no-placeholders
```

Comment:
Why change this?

Reply:
Just to simplify the output a little bit. It's one less new import in the primary module. We newly export `$baz` to make sure it is kept alive in the secondary module via a use in the primary module.